Engines/Wine/QuickScript/GoG Script/script.js

```diff
@@ -0,0 +1,70 @@
+include(["engines", "wine", "quick_script", "quick_script"]);
+include(["utils", "functions", "net", "download"]);
+include(["engines", "wine", "engine", "object"]);
+
+function GogScript() {
+    QuickScript.call(this);
+}
+
+GogScript.prototype = Object.create(QuickScript.prototype);
+
+GogScript.prototype.constructor = GogScript;
+
+GogScript.prototype.gogSetupFileName = function(setupFileName) {
+    this._setupFileName = setupFileName;
+    return this;
+}
+
+/**
+ * Presents a Gog.com login window to the user, login to its account and return a token that can be used later.
+ * Stores the tocken in a parameter
+ * @param {SetupWizard} setupWizard The setupWizard to use
+ * @returns {GogScript} This
+ */
+GogScript.prototype.loginToGog = function (setupWizard) {
+    var browserWindow = setupWizard.createBrowser("Please login to your GoG.com account so that we can download the game for you:");
```

Comment:
In case we support `const` we should use it (I'm not sure if we support it)

Reply:
No we don't...